shared/ui/radio-group/radio-group.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { OptionsRadioGroup, RadioGroup } from '@/shared/ui';
+
+const mockedOptions: OptionsRadioGroup = [
+  {
+    label: 'First',
+    value: '1',
+  },
+  {
+    label: 'Second',
+    value: '2',
+  },
+  {
+    label: 'Third',
+    value: '3',
+  },
+];
+
+const meta = {
+  argTypes: {
+    disabled: {
+      control: 'boolean',
+    },
+    options: {
+      defaultValue: mockedOptions,
+    },
+  },
+  component: RadioGroup,
+  tags: ['autodocs'],
+  title: 'Components/RadioGroup',
```

Comment:
Since the component is stored in the `shared/ui` folder, it makes sense to change the name to `UI/RadioGroup` similar to its siblings.

_Some components have been saved in the `Components/...` folder due to inconsistency between the programmers at launch. This folder is to be deleted and its contents moved to `shared/ui`.